packages/ui/src/council/modals/VoteForCouncil/components/CandidacyReview.tsx

```diff
@@ -33,9 +33,9 @@ export const CandidacyReview = memo(({ candidate, minStake }: CandidacyReviewPro
       <h4>{candidate.info.title}</h4>
 
       <CandidacyPointList>
-        <CandidacyPoint>The More Important the Work, the More Important the Rest</CandidacyPoint>
-        <CandidacyPoint>How to build a loyal community online and offline</CandidacyPoint>
-        <CandidacyPoint>Helping a local business reinvent itself</CandidacyPoint>
+        {candidate.info.bulletPoints.map((bulletPoint, index) => (
+          <CandidacyPoint key={index}>{bulletPoint}</CandidacyPoint>
```

Comment:
`key={index}` is probably not needed here

Reply:
yeah, you are right,
But in my opinion react requires key in the map elements to identify map elements.
In this case, we don`t use index for key, and use something related to bullet points.
Since graphql query returns only string[] for bp array, so I used index for key.
If you think the key is not needed, I can remove it.

Reply:
yeah please do since is not used right now